extensions/conventional-comments/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "conventional-comments",
+  "title": "Conventional comments",
+  "description": "Paste conventional comment into your foremost application",
+  "icon": "icon.png",
+  "author": "krzysztofzuraw",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Conventional comments",
```

Comment:
**suggestion**: Use action form
To express the action of the command, how about using "Insert Conventional Comment" (title cased as well).